### PR TITLE
organize the order of templates view; add cron template and Spark, HDFS templates

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/json/NodeEventView.java [hunk 1]
```diff
@@ -49,6 +49,7 @@
     private String nodeOwner;
     private String defaultJMXUrl;
     private String proactiveJMXUrl;
+    private boolean disabledMonitoring;
```
Comment (fviale) [inline rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/json/NodeEventView.java]: I'm not sure this change belongs to this commit.

File: config/workflows/templates/9/job.xml [hunk 2]
```diff
@@ -1,59 +1,58 @@
...
-# Prepare working directory (For Dataspaces and serialized task file) 
-workspaceHost = localspace 
-workspaceContainer = localspace 
-workspaceVolume = '-v '+localspace +':'+localspace+' ' 
-# Prepare container working directory 
-containerWorkingDirectory = '-w '+workspaceContainer+' ' 
-# Save pre execution command into magic variable 'preJavaHomeCmd', which is picked up by the node 
-preJavaHomeCmd = dockerRunCommand + dockerParameters + proActiveHomeVolume + workspaceVolume + containerWorkingDirectory + containerName 
+              <![CDATA[
+# Prepare Docker parameters
```
Comment (fviale) [inline config/workflows/templates/9/job.xml]: remove the new line after <script

File: config/workflows/templates/9/job.xml [hunk 3]
```diff
@@ -1,59 +1,58 @@
...
-echo '---------------------- Start Task No. 1 ---------------------------------------------------------'
-echo 'This task runs inside a Docker container. It can install new software or change software versions without impacting other software or tasks.'
-echo "Now, lets install git in this container by executing 'apt-get install git -y'."
-echo "We could clone a git repo."
-echo 'Lets pretend we created some statistics about this git repo, we save it in a file (stats.log).'
-echo "In the 'Data Management' tab, the stats.log file is setup to be transferred between tasks."
-echo "This is a statistic, 1,2,3,4,5,6,7,8. Saved in stats.log." > stats.log
+            <![CDATA[
+echo '---------------------- Execute a task inside a Docker Container Example -----------------------------'
+echo '---------------------- Start Task No. 1 ---------------------------------------------------------'
```
Comment (fviale) [inline config/workflows/templates/9/job.xml]: same remark as above

File: config/workflows/templates/9/job.xml [hunk 4]
```diff
@@ -1,59 +1,58 @@
...
+echo 'This task runs inside a Docker container. It can install new software or change software versions without impacting other software or tasks.'
+echo "Now, lets install git in this container by executing 'apt-get install git -y'."
+echo "We could clone a git repo."
+echo 'Lets pretend we created some statistics about this git repo, we save it in a file (stats.log).'
+echo "In the 'Data Management' tab, the stats.log file is setup to be transferred between tasks."
+echo "This is a statistic, 1,2,3,4,5,6,7,8. Saved in stats.log." > stats.log
```
Comment (fviale) [inline config/workflows/templates/9/job.xml]: I don't understand this message, as I don't see this command being executed in the script

File: config/workflows/templates/9/job.xml (hunk 2 above)
Comment (ShatalovYaroslav) [inline config/workflows/templates/9/job.xml]: It was incorrect auto-formatting. The new commit updated that changes

File: config/workflows/templates/9/job.xml (hunk 4 above)
Comment (ShatalovYaroslav) [inline config/workflows/templates/9/job.xml]: It was this message in the original source. I didn't change this part.

**Pull request merged**